packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.types.ts

```diff
@@ -131,15 +131,26 @@ export interface IColorPickerStrings {
   blue?: string;
 
   /**
-   * Label for the alpha text field and slider.
+   * Label for the alpha text field and slider aria-label.
    * @defaultvalue Alpha
    */
   alpha?: string;
 
+  /**
+   * Customized aria-label for the alpha slider.
+   */
+  alphaAriaLabel?: string;
+
   /**
    * Aria label for the hue slider.
    * @defaultvalue Hue
    */
+  hueAriaLabel?: string;
+
+  /**
+   * Aria label for the hue slider.
+   * @deprecated use 'hueAriaLabel'
```

Comment:
Nitpicky thing but it's generally good to use backticks for prop names so they get formatted as code in the docs.
```suggestion
   * @deprecated Use `hueAriaLabel`
```